### ISSUE TITLE
okio for mingwx64 not found

### ISSUE THREAD

**Issue by NikkyAI** [jmfayard/kotlin-cli-starter#17]
seems to be unable to find the okio artifact for windows

```
Execution failed for task ':compileKotlinNative'.
> Could not resolve all files for configuration ':nativeCompileKlibraries'.
   > Could not find com.squareup.okio:okio-mingwx64:3.0.0-alpha.5.
     Searched in the following locations:
       - https://repo.maven.apache.org/maven2/com/squareup/okio/okio-mingwx64/3.0.0-alpha.5/okio-mingwx64-3.0.0-alpha.5.pom
       - https://jcenter.bintray.com/com/squareup/okio/okio-mingwx64/3.0.0-alpha.5/okio-mingwx64-3.0.0-alpha.5.pom
     Required by:
         project : > com.squareup.okio:okio-multiplatform:3.0.0-alpha.5
```

buildscan: https://scans.gradle.com/s/pfmvammsnt6m6

**Comment by jmfayard** [comment]
I forwarded the issue upstream https://github.com/square/okio/issues/934

**Comment by jmfayard** [comment]
will be fixed in alpha.6

**Comment by jmfayard** [comment]
Resolved by upgrading to okio 3.0.0-alpha.6
